src/locale/locales/de/messages.po

```diff
@@ -636,32 +602,28 @@ msgstr "Überprüfe deinen Posteingang auf eine E-Mail mit dem Bestätigungscode
 
 #: src/view/com/modals/Threadgate.tsx:72
 msgid "Choose \"Everybody\" or \"Nobody\""
-msgstr ""
+msgstr "Wählen Sie \"Alle\" oder \"Niemand\""
 
 #: src/view/screens/Settings/index.tsx:697
 msgid "Choose a new Bluesky username or create"
-msgstr ""
+msgstr "Wähle einen neuen Bluesky-Benutzernamen oder erstelle"
```

Comment:
I guess in English this sentence doesn't make sense like it doesn't in German?

Like I would think you need to add "or create one"? In German it becomes very obvious that the sentence is only half finished. Maybe there is more text added afterward that I can't see here?

You could fix it by turning words around so that both actions are positioned in front and reference the new username.

```suggestion
msgstr "Wähle oder erstelle einen neuen Bluesky-Benutzernamen"
```

Reply:
Yes I think this might be a case of where the English text needs fixing (which I'll look into), but for now your suggestion seems to resolve the German translation neatly.